server/src/config.ts

```diff
@@ -141,6 +141,9 @@ export interface SystemConfig {
   reverseGeocoding: {
     enabled: boolean;
   };
+  importFaces: {
```

Comment:
I wonder if it would make sense to put this in a top level `metadata` category. So we could have other exif related settings in the same group. Something like:

```ts
metadata: {
  faces: {
    enabled: boolean; // or import: boolean
  },
  tags: {
    import: boolean;
  }
}
```

Reply:
I like that idea!

Reply:
Ok, will do

Reply:
Implemented by 20946ee0c93058dbb22de391524d38516798a49c

Reply:
Awesome, looks great.